assets/src/edit-story/components/header/buttons/update.js

```diff
@@ -72,23 +81,28 @@ function Update() {
     default:
       text = __('Save draft', 'web-stories');
       return (
-        <Outline
+        <Button
+          variant={BUTTON_VARIANTS.RECTANGLE}
+          type={BUTTON_TYPES.TERTIARY}
+          size={BUTTON_SIZES.SMALL}
           onClick={() => saveStory({ status: 'draft' })}
-          isDisabled={
-            !hasMetaBoxes && (isSaving || isUploading || !hasNewChanges)
-          }
+          disabled={!isEnabled}
         >
           {text}
-        </Outline>
+        </Button>
       );
   }
 
   return (
-    <ButtonWithChecklistWarning
+    <Button
```

Comment:
So, no more checklist warning icon? I assume that's per agreement, I just haven't seen that convo.

Reply:
Oh, is it [this convo](https://github.com/google/web-stories-wp/pull/6277#discussion_r576425810)? I think we need UX approval to remove the icon - just to be sure!

Reply:
[Confirming](https://xwp.slack.com/archives/C01ESBB8ZN1/p1614930022067400) with UX.